### PR TITLE
OCLOMRS-718: Update README link to wiki to point to the actual wiki a…

### PR DESCRIPTION
**Pull request by blackdevelopa**
# JIRA TICKET NAME:
[Update README link to wiki to point to the actual wiki and not the Roadmap document](https://issues.openmrs.org/browse/OCLOMRS-718)

# Summary:
This PR update README link to wiki to point to the actual wiki and not the Roadmap document

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ $ npm run watch:css
 
 ## How to contribute to this project?
  - Please read [OpenMRS wiki](https://wiki.openmrs.org/) for awareness on the code of conduct used in OpenMRS organization 
- - And to get you started with the project please use the OpenMRS-OCL [documentation](https://docs.google.com/document/d/1R_Fgr5SBl4xFNJgj6yMJNVY63b5H_OUqM55o1GFqFKs/edit#heading=h.rc908wooykzg)
+ - And to get you started with the project please use the OpenMRS-OCL [documentation](https://wiki.openmrs.org/display/projects/OCL+Client+for+OpenMRS)
```
Comment (HerbertYiga) [inline README.md]: do we have any information on the added ocl link?

Comment (blackdevelopa) [reply README.md]: Yes. The added OCL link leads to the actual wiki page rather than the project roadmap.

File: README.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ $ npm run watch:css
...
```
Comment (dkayiwa) [inline README.md]: Can you use a short link as tipped here? https://talk.openmrs.org/t/gsod-2019-project-ideas-proposed-projects-deadline-tommorow-23-april-2019-8pm-utc-outstanding-tasks/22940/2

Comment (blackdevelopa) [reply README.md]: Thanks @dkayiwa . I have updated using a short link

File: README.md (hunk 1 above)
Comment (dkayiwa) [reply README.md]: What value do we have for a link which points to wiki page with no information?

Comment (blackdevelopa) [reply README.md]: Kindly check. The page has been updated as well

**Pull request merged**